tracktor/order/send_email_to_customer_when_package_is_in_transit/mesa.json

```diff
@@ -0,0 +1,60 @@
+{
+  "key": "send_email_to_customer_when_package_is_in_transit",
+  "name": "Send Email to customers when their package is \"In Transit\"",
+  "version": "1.0.0",
+  "description": "",
+  "video": "",
+  "readme": "",
+  "tags": [],
+  "source": "tracktor",
+  "destination": "email",
+  "enabled": true,
```

Comment:
I would recommend setting "enabled" to "false". Since this is an email that goes to the customer, I would assume the merchant would want to make changes to this email so it matches their store's business before enabling the workflow.